AppCenterDistribute/AppCenterDistribute/MSDistributePrivate.h

```diff
@@ -225,32 +230,46 @@ static NSString *const kMSTesterAppUpdateSetupFailedKey = @"MSTesterAppUpdateSet
 - (BOOL)handleUpdate:(MSReleaseDetails *)details;
 
 /**
- * Store details about downloaded release to report it after installation.
+ * Save details about a downloaded release.
+ * After an app is updated and restarted, this info will be used to report a download and to update the group ID
+ * (if it was changed).
```

Comment:
Missing `@param`.

Reply:
Sorry, I'm not sure what you mean. Here is the only `details` parameter and it's documented. Maybe I didn't get something. Maybe I don't understand something.

Reply:
Oops.. I didn't see that. It's weird, GitHub didn't show me that in my previous screen. 😄